docs/tutorial.rst

```diff
@@ -1,16 +1,67 @@
 Start Here: ``datascience`` Tutorial
-======
+====================================
 
-In progress.
+This is general introduction to the functionality in
+:py:mod:`datascience`.  For a reference guide, please see
+:ref:`tables-overview`.
 
 Introduction
-------
+------------
+
+First, load the :py:mod:`datascience` module:
+
+.. ipython:: python
+
+   import datascience as ds
+
+In the IPython notebook, type `ds.` followed by the TAB-key to see a list of members.
+The most important of these is the :py:class:`Table` class, which is
+the structure used to represent columns of data.
```

Comment:
Yup, I agree. Since this tutorial will be put online once its merged and students are actively referring to the docs right now, let's write the docs assuming we're going to use `from datascience import *` for now. We can comment out the docs that use `import datascience as ds` for now and then put them back in once this semester finishes up.